Character.cpp

```diff
@@ -86,32 +84,40 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
-
-#include <assert>
-void Character::attackInternal(Character& other)
+void DefeatAndLevelUp(int& startingValue , int& currentValue)
```

Comment:
The convention is to make the first letter a Capital letter when defining static member functions.  This is not a member function of a class, so it doesn't need to follow that same convention.  It should stick to the standard `camelCaseConventionForNamingVariablesAndFunctions`.
Also, spacing around your comma needs fixing.
- [ ] revise